### ISSUE TITLE
Add Check for New Content with Focus Items in the Modal

### ISSUE THREAD

**Issue by andyvanavery31** [NCIOCPL/ncids#2072]
### Issue Description
In the creation of the modal on the Digital Platform, we found that the modal does not recognize any new items that can be focused on for keyboard navigation, and the user cannot keyboard navigate off the 'X' in the corner to interact with the contents of the modal. This ticket will be to update the NCIDS Modal to fix this behavior.

### What's the expected result?
- If there is an item in the modal that can have focus, then the modal will be able to recognize that it exists, navigate to and from it, and use all keyboard navigation commands to interact with it for traceability.

### What's the actual result?
- When new content has focus items in the modal, the modal does not recognize that they are new and there, and the keyboard cannot navigate to the new focus item.

### Related Tickets
* Issue #2054 
* Issue #2067
